homeassistant/components/sonos/number.py

```diff
@@ -34,7 +35,7 @@ async def async_setup_entry(
 ) -> None:
     """Set up the Sonos number platform from a config entry."""
 
-    def available_soco_attributes(speaker: SonosSpeaker) -> list[str]:
+    def available_soco_attributes(speaker: SonosSpeaker) -> SocoFeatures:
         features = []
```

Comment:
```suggestion
        features: SocoFeatures = []
```
`SocoFeatures` could be used here too. Many it should only be moved here (from `.helpers`), not entirely removed.